salt/pillar/elife-bot.sls

```diff
@@ -41,3 +41,30 @@ elife:
     mockserver:
         expectations:
             elife_bot: salt://elife-bot/config/mockserver-elife-bot.sh
+
+    multiservice:
```

Comment:
sure you need all this stuff locally? numbers could easily be smaller, e.g. 2 per process

Reply:
they aren't run locally as processes.sls and multiservice.sls are disabled in the top file. I do rely on the project's formula describing exactly what pillar data it needs so it can be compared to that in builder-private. The numbers are just for demonstration only, happy to drop them to 2 each

Reply:
done